solutions/devsprint-1/DeliveryAppChallenge/Components/Cells/RestaurantCell.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  RestaurantCell.swift
+//  DeliveryAppChallenge
+//
+//  Created by Edson Neto on 04/11/21.
+//
+
+import UIKit
+
+class RestaurantCell: UITableViewCell{
+    static let identifier = "RestaurantCellIdentifier"
+        
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?){
+        super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
+        setup()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private let restaurantLogo: UIImageView = {
+        let logoImageView = UIImageView()
+        logoImageView.image = #imageLiteral(resourceName: "restaurant-logo")
+        logoImageView.translatesAutoresizingMaskIntoConstraints = false
+        logoImageView.contentMode = .scaleAspectFill
+        logoImageView.layer.cornerRadius = 23
+        logoImageView.clipsToBounds = true
+        return logoImageView
+    }()
+    
+    let restaurantNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.boldSystemFont(ofSize: 16)
+        label.textColor = .label
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    let restaurantTypeLabel: UILabel = {
```

Comment:
Como essa constante não está sendo acessada fora da classe ela pode ser privada.
```suggestion
    let restaurantTypeLabel: UILabel = {
```

Reply:
Feito